sdk/python/kubeflow/training/api/training_client_test.py

```diff
@@ -547,6 +561,370 @@ def __init__(self, kind) -> None:
 ]
 
 
+test_data_get_job_conditions = [
+    (
+        "valid flow with default namespace and default timeout",
+        {"name": TEST_NAME},
+        generate_job_with_status(create_job()),
+    ),
+    (
+        "valid flow with failed job condition",
+        {"name": TEST_NAME, "namespace": FAILED},
+        generate_job_with_status(create_job(), namespace=FAILED),
+    ),
+    (
+        "valid flow with restarting job condition",
+        {"name": TEST_NAME, "namespace": RESTARTING},
+        generate_job_with_status(create_job(), namespace=RESTARTING),
+    ),
+    (
+        "valid flow with running job condition",
+        {"name": TEST_NAME, "namespace": RUNNING},
+        generate_job_with_status(create_job(), namespace=RUNNING),
+    ),
+    (
+        "valid flow with created job condition",
+        {"name": TEST_NAME, "namespace": RUNNING},
+        generate_job_with_status(create_job(), namespace=RUNNING),
+    ),
+    (
+        "valid flow with all parameters set",
```

Comment:
Should we combine this test case with `valid flow with default namespace and default timeout` ?
Why do you want to separate them ?